### PR TITLE
Rs 2023.01.20

### PR DESCRIPTION
**Pull request by rschlaikjer**
2023-01-21-130943_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt [hunk 1]
```diff
@@ -1063,57 +1151,66 @@ There isn't a single thing that I can do―――
...
+Slowly but surely,
+ I'll take all those painful memories that tear at your heart,
+ and warp them into something even more miserable―――"
 }
 [sha:f98e30c6187edbe0c59102a6cb7010dd38e36eac]{
 -- Page 64, Offset 35438.
 -- 「ひとかけらの救いもない“思い出”になったら、それが本物なんだって、アナタの過去に組み込むの。
-"Once it turns into a memory devoid of even the slightest trace of hope, only then does it meld with your past, becoming the real thing.
+"Once I've stripped every last shred of hope from these 'memories',
+ I'll merge them back into your past, and they'll become your new reality.
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt:1197]: 救い again but might be hard to render as salvation here. guess we'll leave it be unless you have some good idea

File: script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt [hunk 2]
```diff
@@ -46,12 +50,15 @@ As I sprint toward her, the Dead close in on me from both sides.
...
 [sha:c98bf68e4a3baeefe08fdc5c2474f6343aa6c588]{
 -- Page 5, Offset 35229.
 -- 「あ、ゴメーン！　今はわたしの方が若いんだった☆
-"Ah, my bad! I forgot I'm actually younger than you now ☆
+// Forced linebreak otherwise the ☆ renders on its own awkwardly
+"Oops, silly me!%{n}
+I forgot I'm actually younger than you now ☆
 }
 [sha:fe286acdc6960ada6fa967e872ffa8ff038bdd7a]{
 -- Page 5, Offset 35230.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt:64]: Mixing name + you feels kinda off to me here

File: script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt [hunk 3]
```diff
@@ -86,7 +96,7 @@ Realizing that I'm getting close, Noel immediately leaps back.
 [sha:e75f6601c7418d610a614cf2ba0170eafda813f3]{
 -- Page 7, Offset 35236.
 -- 一瞬で10メートルの後退。
-In an instant, she retreats ten meters backward.
+She retreated ten meters in the blink of an eye.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt:99]: Doesn't sound right to me

File: script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt [hunk 4]
```diff
@@ -86,7 +96,7 @@ Realizing that I'm getting close, Noel immediately leaps back.
...
 }
 [sha:a651a2eff4fb14b22ecb56fce2c32f7e1c9f6e39]{
 -- Page 7, Offset 35237.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt:102]: Feels overly elaborated

File: script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt [hunk 5]
```diff
@@ -96,12 +106,13 @@ In an instant, she retreats ten meters backward.
...
 -- 今の俺なら、あと４秒でその首を切断する。
-As I am right now, I'll be able to sever her neck in about four more seconds.
+As I am right now,
+ it'll only take me another four seconds to slice that head off her shoulders.
 }
 [sha:1258c6a4ed6b016a1cbff562134d61f573c9ee38]{
 -- Page 8, Offset 35239.
 -- 「ああそう、なら―――！」
-"Oh I see. In that case―――!"
+"Alright then, take this―――!"
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt:115]: 'Really now' doesn't seem right to me here... sounds too posh I guess / not direct an answer enough? To me this sentence seems more like an "I get it, not good enough to take you down is it? Alright eat this then".

File: script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt [hunk 6]
```diff
@@ -547,7 +598,10 @@ I sink into the darkness, together with my knife.
 [sha:ed85db55a4e00a881277dbc363f227b88047af4b]{
 -- Page 32, Offset 35328.
 -- 暗闇で声を上げる。
-I raise my voice within the darkness.
+// What the fuck is the word for a surprised noise in English?
+// Do we even have one? It's not quite a yelp, or an exclamation.
+// I guess we mostly encode with the verb - blurt out in surprise?
+My surprise echoes in the darkness.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt:604]: But as like a noun - 'my wha of surprise' makes him sound like Waluigi

File: script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt [hunk 7]
```diff
@@ -827,27 +894,30 @@ Which means I'm―――
...
 -- ヤツの姿も気配もないのは当然だ。
-No wonder I could neither see nor sense her.
+No wonder I couldn't see or sense her.
 }
 [sha:2a97920bdce66b3fd07d1cbf1b3aac4285f2bf77]{
 -- Page 48, Offset 35387.
 -- 俺は今、精神だけがあの女に囚われている。
-Right now, the only part of me that she's taken captive is my mind.
+// だけ kinda odd here in EN
+Right now, my mind is being held captive by that woman.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt:915]: I mean yeah I stand by my comment

File: script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt (hunk 6 above)
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt]: I guess gasp of surprise actually

File: script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt:64]: Yeah, I thought that at first, but then I was like, you know people kinda talk like this when they're doing the thing in the first clause for effect. But sure, I guess you could just change you to he

File: script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt:101]: Hmmm... sure.
I felt like it linked pretty poorly, as it is often the case with these brief sentences, but maybe it's not a big deal.

File: script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt:98]: Yeah, this is probably wrong, on second thought.

File: script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt (hunk 5 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt]: It's the "sarcastic" ああそう, that's why I chose "really now". She doesn't actually mean, "I get it". I think she's just expressing her frustration, essentially.

File: script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt (hunk 7 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt:910]: Yeah, it doesn't help when I forget a noun. That said, it's an important だけ here. Maybe can be solved with some adjectives.

>Right now, my separated mind is being held captive by her.

**Pull request merged**